ProjectLighthouse.Servers.Website/Pages/SlotSettingsPage.cshtml

```diff
@@ -52,6 +52,37 @@ function onSubmit(){
                         <input type="hidden" name="avatar" id="avatar-encoded">
                     </div>
                 </div>
+
+                <div class="ui divider"></div>
+                <div class="ui red label">
+                    <i class="lock icon"></i>
+                    Locked
+                    <input type="checkbox" name="locked" id="locked" style="margin-left: 5px;">
+                </div>
+                <div class="ui green label">
+                    <i class="check icon"></i>
+                    Copyable
```

Comment:
```suggestion
                    <label for="copyable">
                        <i class="icon copy outline"></i>
                        Copyable
                    </label>
```
Same as above. I think the copyable button having a checkmark is a little ambiguous, somehting like a traditional copy icon might work better.